Code/GraphMol/FindStereo.cpp

```diff
@@ -385,6 +385,64 @@ inline std::string getAtomCompareSymbol(const Atom &atom) {
   return std::to_string(atom.getIsotope()) + atom.getSymbol() +
          std::to_string(atom.getFormalCharge());
 }
+
+bool are_stereobond_controlling_atoms_dupes(
```

Comment:
The style here is inconsistent with the rest of the RDKit, which doesn't use underscores.
It's not the end of the world since this is internal code, but I would rather not start mixing styles like this.

Reply:
Ok, will do.